internal/tui/components/dialogs/commands/commands.go

```diff
@@ -281,6 +281,7 @@ func (c *commandDialogCmp) defaultCommands() []Command {
 			ID:          "switch_model",
 			Title:       "Switch Model",
 			Description: "Switch to a different model",
+			Shortcut:    "alt+m",
```

Comment:
I couldn't trigger this at all on Rio on macOS.

Neither <kbd>control+m</kbd>, <kbd>option+m</kbd> or <kbd>command+m</kbd> worked. Any tips?

Perhaps you just meant <kbd>control+m</kbd>? All similar commands use <kbd>control</kbd>.

Reply:
ctrl+m == enter

it won't work

Reply:
Alt (i.e. option) a dicey one because by default it's disabled in _many_ macOS terminals. 

Can we bind to <kbd>ctrl+m</kbd> when disambiguate keys is supported and fall back to <kbd>ctrl+l</kbd> otherwise? L is for modeL.